dapps/tezosDomains.tsx

```diff
@@ -401,7 +401,7 @@ export function tezosDomains(
                 action: BUY_ADDRESS.name,
                 description: (
                   <p>
-                    TzSafe doesn't support the entrypoint:{" "}
```

Comment:
You don't need spaces anymore? I am wondering if we can use template string :thinking:

Reply:
Yes you're right :-)

Reply:
I updated all of occurrences of this string